File: templates/tests-go/BREACH-001.go
package compliance

import (
	"testing"
)

func TestEnforceDataRetentionPolicy(t *testing.T) {
	config := GetConfig()
	retention := config.DataLifecycle.Retention
	if retention <= 0 || retention > 365 {
		t.Errorf("Data retention policy must be between 1 and 365 days, got %d", retention)
	}
}

File: templates/tests-python/BREACH-001.py
import pytest


def test_enforce_data_retention_policy():
    """Verify data retention policy is configured and reasonable."""
    config = get_config()
    assert "data_lifecycle" in config
    assert "retention" in config["data_lifecycle"]
    retention_days = int(config["data_lifecycle"]["retention"])
    assert retention_days > 0
    assert retention_days <= 365
